leaguebot/models/history.py

```diff
@@ -360,7 +365,16 @@ def process_all_pending_battles_once():
         else:
             latest_tick = 0
 
-        battle_data = process_room(room_name, latest_tick)
+        try:
+            battle_data = process_room(room_name, latest_tick)
+        except TooManyRequestsError:
+            logger.exception("Error updating room, attempting to sleep 20 seconds.")
+            time.sleep(20)
```

Comment:
I'd rather it just skip instead of delaying for 20 seconds- I worry that if too many rooms give this error it'll result in massive delays as we pause for 20 seconds for each of them.